src/Cms/Auth.php

```diff
@@ -493,23 +479,19 @@ public function status($session = null, bool $allowImpersonation = true)
     }
 
     /**
-     * Validates the user credentials and returns the user object on success;
-     * otherwise logs the failed attempt
+     * Ensure that email addresses with IDN domains are in Unicode format
+     * and that rate limit was not exceeded
      *
      * @param string $email
-     * @param string $password
-     * @return \Kirby\Cms\User
+     * @return string
      *
-     * @throws \Kirby\Exception\PermissionException If the rate limit was exceeded or if any other error occurred with debug mode off
-     * @throws \Kirby\Exception\NotFoundException If the email was invalid
-     * @throws \Kirby\Exception\InvalidArgumentException If the password is not valid (via `$user->login()`)
+     * @throws \Kirby\Exception\PermissionException If the rate limit was exceeded
      */
-    public function validatePassword(string $email, string $password)
+    public function validateEmail(string $email): string
```

Comment:
Should we make this method protected since it is login specific? 🤔

Reply:
I actually thought about that - and probably yes. In the end, I followed `validatePassword` but maybe there is a reason for this one needed to be public?

Reply:
@lukasbestle will know best

Reply:
I agree we should make it `protected`. It's quite specific and of little use for devs as it doesn't fully validate the email (e.g. it doesn't check whether the user exists).

`validatePassword` is `public` because of use cases where you want to check a password without logging in (e.g. for logging in to third-party sites).